### PR TITLE
 Create example 1 in pass props

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Finish challenge 1 in pass props
 
 https://react.dev/learn/passing-props-to-a-component#challenges
 
 
image

### CODE REVIEW

File: examples/first-component/src/components/Scientist/Scientist.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import Person from "../../types/Person";
+
+interface ScientistProps {
+  scientist: Person;
+}
+
+const Scientist = ({ scientist }: ScientistProps) => {
+  const { imageId, name, profession, awards, discoveries, imageSize } =
+    scientist;
```
Comment (hungdo-agilityio) [inline examples/first-component/src/components/Scientist/Scientist.tsx]: Please only pass props that are necessary for the component. For example, in this case
```typescript
interface ScientistProps {
   imageId: string,
   name: string,
   ...
}

const Scientist = ({imageId, name, ...rest}: ScientistProps) => 
```

File: examples/first-component/src/App.tsx [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+      "https://khunganhonline.com/images/frame/frame_icon/khung-anh-hoa-hinh5aec7498e4003.jpg",
+    name: "Katsuko Saruhashi",
+    profession: "geochemist",
+    awards: `Miyake Prize for geochemistry, Tanaka Prize`,
+    discoveries: "a method for measuring carbon dioxide in seawater",
+    imageSize: 70,
+  },
+];
 
 function App() {
```
Comment (hungdo-agilityio) [inline examples/first-component/src/App.tsx]: Can we use ES6 syntax here

File: examples/first-component/src/types/Person.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  id: string;
+  imageId: string;
+  name: string;
+  profession: string;
+  awards: string;
+  discoveries: string;
+  imageSize: number;
+}
+
+export default  Person;
```
Comment (hungdo-agilityio) [inline examples/first-component/src/types/Person.ts]: Please recheck the format here, has extra spacing

File: examples/first-component/src/types/Person.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+interface  Person {
```
Comment (hungdo-agilityio) [inline examples/first-component/src/types/Person.ts]: Why not we export directly this interface instead of export default

Comment (thanhnguyen-agilityio) [reply examples/first-component/src/types/Person.ts]: For this comment, do you see the warning or error on code?

File: examples/first-component/src/components/Scientist/Scientist.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply examples/first-component/src/components/Scientist/Scientist.tsx]: Fixed

File: examples/first-component/src/App.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply examples/first-component/src/App.tsx]: Fixed

File: examples/first-component/src/types/Person.ts (hunk 4 above)
Comment (thanhtrantruong47) [reply examples/first-component/src/types/Person.ts]: Fixed and I don't see errors or warnings there.

File: examples/first-component/src/components/Scientist/Scientist.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+// https://react.dev/learn/passing-props-to-a-component#challenges
```
Comment (hungdo-agilityio) [inline examples/first-component/src/components/Scientist/Scientist.tsx]: Please help update the naming of this file into `index.tsx`

Comment (hungdo-agilityio) [reply examples/first-component/src/components/Scientist/Scientist.tsx]: Same as ScientistList

Comment (thanhtrantruong47) [reply examples/first-component/src/components/Scientist/Scientist.tsx]: Fixed

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**